app/assets/src/components/views/discovery/table_renderers.scss

```diff
@@ -209,14 +217,35 @@
     }
   }
 
+  .itemRightPane {
+    flex: 1 0 auto;
+
+    .itemName {
+      @include loadingAnimation;
```

Comment:
I'm not sure these loading animations are actually showing up for some reason

![Screen Shot 2019-09-19 at 5 49 52 PM](https://user-images.githubusercontent.com/5652739/65290902-f304ac80-db05-11e9-82fc-8c086407880f.png)

Reply:
Done.